src/__tests__/updater.js

```diff
@@ -0,0 +1,133 @@
+import { enhanceReducer } from "../updater";
+import { OFFLINE_STATUS_CHANGED, RESET_STATE } from "../types";
+import {
+  OFFLINE_COMPLETE_RETRY,
+  OFFLINE_SCHEDULE_RETRY,
+  PERSIST_REHYDRATE,
+  OFFLINE_BUSY
+} from "../constants";
+
+const initialState = {
+  busy: false,
+  lastTransaction: 0,
+  online: false,
+  outbox: [],
+  retryCount: 0,
+  retryScheduled: true,
+  netInfo: {
+    isConnectionExpensive: null,
+    reach: "NONE"
+  }
+};
+
+const config = {
+  queue: {
+    enqueue: jest.fn(),
+    dequeue: jest.fn()
+  },
+  offlineStateLens: (state: any) => {
```

Comment:
Again, should be fine to rely on the default config for this stuff, no need to mock